src/components/WalletConnect/NanoContract/NanoContractMethodArgs.js

```diff
@@ -129,6 +137,57 @@ export const NanoContractMethodArgs = ({ blueprintId, method, ncArgs }) => {
   )
 };
 
+/**
+ * Component resposible to render the appropriate format for the value
+ * taking in consideration the type.
+ *
+ * Remarks
+ * The values received in here when derived from 'byte' type like
+ * 'TxOutputScript', 'TokenUid' and 'VertexId' are already in its
+ * hexadecimal format.
+ *
+ * Values of type 'Address', which also derives from 'byte' are
+ * in its base58 format.
+ *
+ * Values of type 'SignedData[Result]' arrives here using its data
+ * part already.
+ *
+ * @param {Object} props
+ * @param {string} props.type An argument type
+ * @param {string} props.value An argument value
+ */
+const ArgValue = ({ type, value }) => {
+  const network = useSelector((state) => new Network(state.networkSettings.network));
+  if (type === 'Amount') {
+    return renderValue(value);
+  }
+
+  if (type === 'Timestamp') {
+    return getTimestampFormat(value);
+  }
+
+  if (type === 'TxOutputScript') {
+    const parsedScript = parseScriptData(value, network);
+    if (parsedScript && parsedScript.getType() === 'data') {
+      return parsedScript.data;
+    }
+
+    if (parsedScript) {
+      return parsedScript.address.base58;
+    }
+  }
+
+  if (type === 'TokenUid') {
+    if (value === DEFAULT_TOKEN.uid) {
+      return DEFAULT_TOKEN.symbol;
+    }
+    // XXX: We may want to extend to show the token symbol of
+    // registered tokens.
```

Comment:
I think we should do it. What we can do is to show: `tokenUid (tokenSymbol)` and the token symbol only if the token is registered, or ir the default token. 

What do you think?

Reply:
I have added it at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/537/commits/3d5f6f6cf7d6b09139519d3a18bdcc82f76d5881